src/tools/react-native.ts

```diff
@@ -1,4 +1,6 @@
-import { GluegunToolbox } from "gluegun"
+import { GluegunToolbox, filesystem } from "gluegun"
+
+// export const isWindows = process.platform === "win32"
```

Comment:
Can we completely remove this?

Reply:
Turns out, no. We need to keep the `file:///` with the new CLI. Working on it!